kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/system/SystemServiceImpl.java

```diff
@@ -179,7 +179,7 @@ public Object run() {
             // if not found, look for such file in the kura.home directory
             Properties kuraCustomProps = new Properties();
             String kuraCustomConfig = System.getProperty(KURA_CUSTOM_CONFIG);
-            String kuraCustomProperties = IOUtil.readResource(KURA_CUSTOM_PROPS_FILE);
+            String kuraCustomProperties = readResource(KURA_CUSTOM_PROPS_FILE);
```

Comment:
Why not having that as a static import instead of a new method that only wraps IOUtil.readResource?